packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -308,12 +308,13 @@ async function createReportsGroupedByProject(periodId, tenantId, dateFormat = RE
         projectLogger.debug('populating row from records in project');
 
         // set values for columns that are common across all records of projectId
-        const lastProjectRecord = projectRecords[projectRecords.length - 1];
+        const projectSheetRecords = projectRecords.filter((record) => Object.keys(EXPENDITURE_CATEGORIES).includes(record.type));
+        const lastProjectRecord = projectSheetRecords[projectSheetRecords.length - 1];
```

Comment:
Can this ever be null, or is the null checking below for a type checker?

Reply:
`lastProjectRecord` can indeed be null!

It will only happen if someone uploads a spreadsheet with expenditures or awards that reference a project ID that hasn't been defined, but staging does have a test file on it that simulates that scenario, so I assume it happens.

I do think there are at least two options for handling that.  We could skip the row entirely, or we can show whatever information we do have.  I suspect more information is more useful than less, but that's probably something we could confirm with partners (or iterate on).